src/main/java/baseball/game/number/Digit.java

```diff
@@ -0,0 +1,35 @@
+package baseball.game.number;
+
+import java.util.Objects;
+
+public class Digit {
+    public final int digitValue;
+
+    public Digit(int digitValue) {
+        validateDigitValue(digitValue);
+        this.digitValue = digitValue;
+    }
+
+    private void validateDigitValue(int number) {
+        if (number < 1 || number > 9) {
```

Comment:
1과 9 같은 상수들도 상수화 하면서 명확한 네이밍을 해주면 이 숫자의 의미를 더 파악하기 쉬워져서 가독성이 올라갈 것 같아요! :)

Reply:
@tjdtls690 저 생각을 해봤는데, 의미를 파악하는 데에는 숫자의 의미가 불분명하기 보다는 조건식이 의미하는 바가 한눈에 보이지 않아서 그런것 같았어요.
위에서 알려주신 방법처럼 한번에 메서드를 빼는게 더 나은 것 같아서 이렇게 해봤어요! 감사합니다..

```suggestion
        if (isNotOneDigitNumber(number)) {
```

Reply:
사실 메서드로 분리하면서 네이밍을 명확히 해준다면, 그것으로도 가독성이 많이 올라갈 것이라고 생각합니다. 전 개인적으로 좀 심할 정도로 가독성에 대한 강박증이 있어서 그런지, 메서드로 분리해준다고 해도 매직넘버에 해당되는 녀석들은 전부 상수화를 통해 네이밍 처리를 해줍니다..ㅋ;

가영님이 보시기에 메서드 분리만으로도 가독성이 충분하다면, 그것 또한 한 가지의 길이 될 수 있다고 생각합니다.